src/status_im/pairing/core.cljs

```diff
@@ -66,11 +66,10 @@
   (sort (partial compare-installation our-installation-id) installations))
 
 (defn pair-installation [cofx]
-  (let [fcm-token         (get-in cofx [:db :notifications :fcm-token])
-        installation-id (get-in cofx [:db :multiaccount :installation-id])
+  (let [installation-id (get-in cofx [:db :multiaccount :installation-id])
         installation-name (get-in cofx [:db :pairing/installations installation-id :name])
         device-type     utils.platform/os]
-    (protocol/send (transport.pairing/PairInstallation. installation-id device-type installation-name fcm-token) nil cofx)))
```

Comment:
@cammellos what needs to be done on go side to remove fcm tokens and device infos? I think there is also some firebase stuff on go side?